private/model/api/operation.go

```diff
@@ -71,7 +73,11 @@ const op{{ .ExportedName }} = "{{ .Name }}"
 //    if err == nil { // resp is now filled
 //        fmt.Println(resp)
 //    }
+{{ $blacklisted := isBlacklistedService .API -}}
+{{ if not $blacklisted -}} 
 //
+// Please also see ` + redirectsURL + `/goto/WebAPI/{{ $.API.Metadata.UID }}/{{ .ExportedName }}
```

Comment:
`redirectURL` should be added to the template data not injected into the string like this.